tldw_Server_API/tests/Audio/test_parakeet_core_buffer_and_metadata.py

```
@@ -0,0 +1,174 @@
+def test_variant_decode_selection(monkeypatch):
+    # Create dummy modules for each variant path
+    mod_nemo = types.ModuleType("Audio_Transcription_Nemo")
+    def _tx_parakeet(audio_np, sample_rate, variant="standard"):
+        return f"nemo:{variant}"
+    mod_nemo.transcribe_with_parakeet = _tx_parakeet
+    sys.modules[
+        "tldw_Server_API.app.core.Ingestion_Media_Processing.Audio.Audio_Transcription_Nemo"
+    ] = mod_nemo
+
+    mod_onnx = types.ModuleType("Audio_Transcription_Parakeet_ONNX")
+    mod_onnx.transcribe_with_parakeet_onnx = lambda audio_np, sample_rate: "onnx:ok"
```

Ruff (0.14.2)

Unused lambda argument: `audio_np`

(ARG005)